drivers/hmis/app/models/hmis/hud/event.rb

```diff
@@ -15,6 +15,7 @@ class Hmis::Hud::Event < Hmis::Hud::Base
   belongs_to :client, **hmis_relation(:PersonalID, 'Client')
   belongs_to :user, **hmis_relation(:UserID, 'User'), inverse_of: :events
   belongs_to :data_source, class_name: 'GrdaWarehouse::DataSource'
+  has_one :project, through: :enrollment
```

Comment:
Why is this added, and is it used anywhere here?
I'm wondering because this relation will return `nil` for events rendered at WIP enrollments. So, we should add a comment about that at least, and be mindful when using.

Reply:
This was added so that permissions can resolve through the project, but good point on the WIP issue. I just added another case in the loader to use the enrollment directly instead of doing it this way.